### PR TITLE
format & fix typo

### PR DESCRIPTION
**Pull request by atefBB**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Browser (Add scripts in HTML)
 <script type="text/javascript" src=".../re-tree.js"></script>
 <script type="text/javascript" src=".../ng-device-detector.js"></script>
 ```
-Adding in module dependencies
+Adding to you app module dependencies
```
Comment (srfrnk) [inline readme.md]: "your" ?

Comment (atefBB) [inline readme.md]: fix typo in 'fix typo' PR ^_____^

**Comment by atefBB** [comment]
@srfrnk DONE !

**Comment by atefBB** [comment]
Yoooooooooooshaaaaa!
thx !

**Comment by srfrnk** [comment]
sure - enjoy :)

**Pull request merged**